data/sql/updates/pending_db_world/arena-rewards.sql

```diff
@@ -0,0 +1,72 @@
+DELETE FROM `arena_season_reward_group` WHERE `arena_season` IN (1, 2, 3, 4);
+INSERT INTO `arena_season_reward_group` (`id`, `arena_season`, `criteria_type`, `min_criteria`, `max_criteria`, `reward_mail_template_id`, `reward_mail_subject`, `reward_mail_body`, `gold_reward`) VALUES
+-- Season 4 (Brutal)
+(21, 4, 'abs', 1, 1, 0, '', '', 0),
+(22, 4, 'pct', 0, 0.5, 262, '', '', 0),
+(23, 4, 'pct', 0.5, 3, 0, '', '', 0),
+(24, 4, 'pct', 3, 10, 0, '', '', 0),
+(25, 4, 'pct', 10, 35, 0, '', '', 0),
+-- Season 3 (Vengeful)
+(26, 3, 'abs', 1, 1, 0, '', '', 0),
+(27, 3, 'pct', 0, 0.5, 211, '', '', 0),
+(28, 3, 'pct', 0.5, 3, 0, '', '', 0),
+(29, 3, 'pct', 3, 10, 0, '', '', 0),
+(30, 3, 'pct', 10, 35, 0, '', '', 0),
+-- Season 2 (Merciless)
+(31, 2, 'abs', 1, 1, 0, '', '', 0),
+(32, 2, 'pct', 0, 0.5, 182, '', '', 0), -- Mail needs verification
+(33, 2, 'pct', 0.5, 3, 0, '', '', 0),
+(34, 2, 'pct', 3, 10, 0, '', '', 0),
+(35, 2, 'pct', 10, 35, 0, '', '', 0),
+-- Season 1
+(36, 1, 'pct', 0, 0.5, 182, '', '', 0), -- Mail needs verification
+(37, 1, 'pct', 0.5, 3, 0, '', '', 0),
+(38, 1, 'pct', 3, 10, 0, '', '', 0),
+(39, 1, 'pct', 10, 35, 0, '', '', 0);
+
+DELETE FROM `arena_season_reward` WHERE `group_id` IN (1, 6, 11, 16, 21, 22, 22, 23, 24, 25, 26, 27, 27, 28, 29, 30, 31, 32, 32, 33, 34, 35, 36, 36, 37, 38, 39);
+INSERT INTO `arena_season_reward` (`group_id`, `type`, `entry`) VALUES
+-- Season 8 (Wrathful)
+(1, 'achievement', 4599),
+-- Season 7 (Relentless)
+(6, 'achievement', 3758),
+-- Season 6 (Furious)
+(11, 'achievement', 3436),
+-- Season 5 (Deadly)
+(16, 'achievement', 3336),
```

Comment:
CC: @walkline, was all four of these seasons using the [Deadly Gladiator achievement](https://github.com/azerothcore/azerothcore-wotlk/pull/19858/files#diff-353c1135e51c7fb21c471acdeb028d0debce5652a038c9a9c587ba74bbc12c4eR55-R76) an oversight or am I missing something?

Reply:
Most likely, I didn't have the data for the older seasons, so I just used the same values.